src/data/contributors/Ankush Lohani.json

```diff
@@ -0,0 +1,11 @@
+{
+    "salutation": "HOLA GEEKS",
+    "name": "Ankush Lohani",
+    "skills": [HTML CSS C JS],
```

Comment:
Put Skills in double quotes like this -> "HTML"

Reply:
done